help_to_heat/portal/download_views.py

```diff
@@ -138,6 +197,26 @@ def download_feedback_view(request):
     return response
 
 
+@require_http_methods(["GET"])
+@decorators.requires_service_manager
+def download_referrals_all_view(request):
+    feedbacks = portal_models.Referral.objects.all()
```

Comment:
Was just looking at the PR for general learning, but I think this variable might want to be named `referrals` as opposed to `feedbacks` in this function and the one below?